### DISCUSSION TITLE
Updating container config

### DISCUSSION THREAD

**Discussion by eth0h4ckr** [mbround18/valheim-docker#98] [Q&A] [answered]
Docker noob here. What is the proper way to deploy/update the container config once its running? Lets say I want to enable auto updates. What is the proper way to do that?

**Comment by mbround18** [comment, marked as answer]
are you using docker compose? or straight docker?

**Reply by eth0h4ckr** [reply]
docker-compose

**Reply by mbround18** [reply]
You can add the environment variables like below to auto update the game:

```
environment: 
  - AUTO_UPDATE=1
  - AUTO_UPDATE_SCHEDULE="0 1 * * *"
```

Occasionally you will also need to update the container by running `docker-compose pull valheim`

**Reply by eth0h4ckr** [reply]
So if i change the vars in the docker-compose file after already deploying the container, I just run docker-compose pull valheim ? My understanding is that will just update the image? Maybe Im not being clear in what im asking. If I already have the container deployed/running after deploying it with docker-compose, how do I update the environment variables? Sorry, Im pretty green when it comes to docker.

**Reply by mbround18** [reply]
`docker-compose pull valheim` will update the image to include new features <3

You can change env variables at any time and they will update what functionality is enabled in the server even after youve already deployed it :) though you will have to restart your docker compose after you change env variables.

**Reply by eth0h4ckr** [reply]
Got ya. Thanks for the help!
